src/bp/core/botpress.ts

```diff
@@ -121,14 +121,16 @@ export class Botpress {
   }
 
   async checkJwtSecret() {
-    let jwtSecret = this.config!.jwtSecret
-    if (!jwtSecret) {
-      jwtSecret = nanoid(40)
-      this.configProvider.mergeBotpressConfig({ jwtSecret })
+    // @deprecated : botpress > 11 remove jwtSecret
+    // @ts-ignore
```

Comment:
Deprecate message isn't very clear, we don't remove jwtSecret, we just rename it appSecret